### ISSUE TITLE
I suggest using OpenBSD's ksh instead of mksh

### ISSUE THREAD

**Issue by Curculigo** [iglunix/iglunix#124]
https://github.com/ibara/oksh

**Comment by Ella-0** [comment]
No. We specifically use mksh for the high compatibility with bash.

**Comment by Curculigo** [comment]
> No. We specifically use mksh for the high compatibility with bash.

I suggested oksh because its high compatibility with bash! My experience with mksh and oksh is mostly on MSYS2. I found oksh is more compatible with bash than mksh. You can even use the same PS prompt syntax like bash!
